src/Variant.js

```diff
@@ -0,0 +1,17 @@
+const Variant = (ctor, ...args) => {
+    return {[ctor]: [...args]}
+}
+
+const Some = (args) => {
+    return Variant('Some', args)
+}
+
+const None = () => {
+    return Variant('None')
+}
```

Comment:
This won't be needed after implementing #73

Reply:
That won't be implemented